CallAutomation_SimpleIvr/Program.cs

```diff
@@ -15,9 +15,12 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
-
 var client = new CallAutomationClient(builder.Configuration["ConnectionString"]);
-var callbackUriBase = builder.Configuration["CallbackUriBase"];
+var baseUri = Environment.GetEnvironmentVariable("VS_TUNNEL_URL")?.TrimEnd('/');
```

Comment:
This will work only if we run locally right? 
What happens if we deploy the application on cloud?

We should add a condition if its local run then use the dev-tunnel-url else fetch it from the config?

Reply:
Now, i think we should also update this for the Appointment reminder application

Reply:
Ohh i see you already have handle it below. Resolving